### PR TITLE
New search menu item

### PR DESCRIPTION
**Pull request by chasulin**
[Figma mocks](https://www.figma.com/file/JGqemgZwlG9e5DQbZpdKgy/Homepage-Above-the-fold?node-id=68%3A98)

### CODE REVIEW

**Review by pnavarrc — APPROVED** [review]
Added some notes, but looks good!

File: src/components/Search/NewMenuItem/NewMenuItem.style.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+export const Subcopy = styled.span`
+  font-size: 0.875rem;
+  color: ${COLOR_MAP.GRAY_BODY_COPY};
+`;
+
+export const Icon = styled(FiberManualRecordIcon)<{ fillColor: string }>`
+  color: ${({ fillColor }) => fillColor};
+  font-size: 1.1rem;
+  margin-right: 14px;
+`;
```
[the remark this answers is not in the document]
Comment (chasulin) [reply src/components/Search/NewMenuItem/NewMenuItem.style.tsx:23]: oh yes good call! renaming the prop, and will add a todo to extract CircleIcon later on

File: src/components/Search/NewMenuItem/NewMenuItem.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  Icon,
+  InnerContent,
+  ArrowIcon,
+} from './NewMenuItem.style';
+import { Region, MetroArea, County } from 'common/regions';
+import { formatEstimate } from 'common/utils';
+import { getLocationIconFillColor } from 'components/Search';
+import { StyledRegionName } from 'components/SharedComponents';
+
+const NewMenuItem: React.FC<{ region: Region; zipCodeInput: string }> = ({
```
Comment (ghop02) [inline src/components/Search/NewMenuItem/NewMenuItem.tsx:15]: Can you add a note to remove New from the name when we retire the current MenuItem component?

Comment (chasulin) [reply src/components/Search/NewMenuItem/NewMenuItem.tsx:15]: theres an item in my homepage checklist for this! will remove the old `MenuItem` and move+rename this one

**Pull request merged**